Claw V1

Calstar9000 opened the pull request.

Mexican1oreo committed:
init commit

Mauric3Moss committed:
Install Roadrunner

Calstar9000 committed:
Claw servo V1

Calstar9000 committed:
Claw servo V1

Calstar9000 committed:
Claw servo V1

Calstar9000 committed:
Claw servo V1

Soren6 commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java:
```diff
@@ -2,18 +2,22 @@
 
     @Override
     public void loop() {
-
```
I dont know if the control scheme has been finalized yet, if so the controls are fine, I would assume that these would be on the operator controller as well, but if it is just temporary for testing, make sure to add a comment indicating so.

samprogram7 commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoClaw.java:
```diff
@@ -0,0 +1,26 @@
+    public Servo servoClaw;
+
+    public void runServoClaw(boolean a, boolean b) {
+
```
wrong

Mexican1oreo commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoWrist.java:
```diff
@@ -0,0 +1,26 @@
+
+public class servoWrist {
+    public Servo servoWrist;
+
```
Yeah, I agree with Soren that a lot of things have been done right. With spacing its kind of hard to tell on GitHub, however we do use tabs rather than spaces when indenting code. Otherwise like Soren said spacing is nice and consistant

Mauric3Moss replied:
Agree with Tony, use tabs and be liberal with line breaks for easy readability.

samprogram7 replied:
Agree with Abbot as well on this one

Calstar9000 committed:
Claw servo V1.1

Calstar9000 committed:
Claw v2

Mexican1oreo commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/constants.java:
```diff
@@ -0,0 +1,8 @@
+package org.firstinspires.ftc.teamcode;
+
+public interface constants {
+    double clawOpenValue = 1;
```
I would call these variables openClawPosition or openClawValue. Something along those lines as it looks better and is a bit more readable

Mexican1oreo commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoClaw.java:
```diff
@@ -9,14 +9,9 @@
-        else if(closeClaw){
-            servoClaw.setPosition(clawCloseValue); //closed
-        }
+    public void runServoClaw(double clawInput) {
```
Change the name of the class to something more like setClawServoPosition or something along those lines as it explains the purpose of the code alot quicker

Mexican1oreo commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/servoWrist.java:
```diff
@@ -9,14 +9,9 @@
-        else if(wristIn){
-            servoWrist.setPosition(wristCloseValue); // closed
-        }
+    public void runServoWrist(double wristInput) {
```
Same issue as the servoClaw class

Calstar9000 committed:
Renamed class, variable, and function name to be more readable

Claw and ClawWrist classes

openClawValue, closedClawValue, openWristValue, and closedWristValue variables

setServoClawPos and setServoWristPos functions

Calstar9000 committed:
Renamed class, variable, and function name to be more readable

Claw and ClawWrist classes

openClawValue, closedClawValue, openWristValue, and closedWristValue variables

setServoClawPos and setServoWristPos functions

Calstar9000 committed:
Renamed class, variable, and function name to be more readable

Claw and ClawWrist classes

openClawValue, closedClawValue, openWristValue, and closedWristValue variables

setServoClawPos and setServoWristPos functions

Calstar9000 commented:
also moved init statements up

ashtrogen commented on TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Claw.java:
```diff
@@ -0,0 +1,21 @@
+
+    }
+
+}
```
make it better, trust :3 <3
